assets/src/edit-story/components/helpCenter/index.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useState } from 'react';
+import { useFeatures } from 'flagged';
+import styled, { ThemeProvider } from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import { theme as dsTheme, ThemeGlobals } from '../../../design-system';
+import { Toggle } from './toggle';
+
+const Wrapper = styled.div`
+  position: absolute;
+  bottom: 16px;
+  left: 11px;
+  z-index: 10;
+
+  @media ${({ theme }) => theme.breakpoint.tablet} {
+    bottom: 24px;
+    left: 23px;
+  }
+
+  @media ${({ theme }) => theme.breakpoint.destop} {
+    bottom: 16px;
+    left: 31px;
+  }
+`;
+
+export const HelpCenter = () => {
+  const { enableQuickTips } = useFeatures();
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    enableQuickTips && (
+      <ThemeProvider theme={dsTheme}>
+        <ThemeGlobals.OverrideFocusOutline />
```

Comment:
the dashboard and edit-story have different `app/index` and the dashboard's getting the new design system implemented on the whole while the editor won't for another release cycle. So since the help center is using the new designs it makes sense to have the theme and the focus outline override here for just the scope of this feature.  

It's this kinda weird limbo space where we're trying to not regress designs but move out of the existing structure.